update.mjs

```diff
@@ -100,6 +100,11 @@ const linters = [
   'xo.config.*',
 ]
 
+// @keep-sorted
+const logs = [
```

Comment:
This is used?

Reply:
Woops, that was a different thing. It's not used